packages/ui/src/theme/createTheme.ts

```diff
@@ -98,9 +98,15 @@ export function createTheme(
       }
       if ('colorMode' in override) {
         cssText += `\n@media (prefers-color-scheme: ${override.colorMode}) {
-          [data-amplify-theme="${name}"][data-amplify-color-mode="system"] {\n${customProperties}\n}
+          [data-amplify-theme="${name}"][data-amplify-color-mode="system"] {
+            ${customProperties}
+            color-scheme: ${override.colorMode};
```

Comment:
Fixes an issue where Chrome doesn't pickup the system color theme, and the "date" button is the wrong color.

Before:
<img width="295" alt="CleanShot 2023-08-28 at 17 24 17@2x" src="https://github.com/aws-amplify/amplify-ui/assets/6165315/75240cc8-0386-481c-9014-723d41426862">

After:
![Uploading CleanShot 2023-08-28 at 17.24.51@2x.png…]()